pkg/apis/devconsole/v1alpha1/component_types_labels.go

```diff
@@ -36,3 +36,9 @@ func (c *Component) GetLabelInstance() string {
 func (c *Component) GetLabelVersion() string {
 	return c.Labels["app.kubernetes.io/version"]
 }
+
+// GetLabelDeploymentConfig retrieves the label deploymentconfig. This would be
+// same as value of label instance.
+func (c *Component) GetLabelDeploymentConfig() string {
```

Comment:
Why not call `GetLabelInstance` directly?

Reply:
@baijum I had discussion with @tinakurian and we have decided not to separate method just to get deploymentconfig label. We are going to use `GetLabelInstance` directly.